jwt/src/test/java/me/uport/sdk/jwt/JWTToolsJVMTest.kt

```diff
@@ -26,12 +28,14 @@ class JWTToolsJVMTest {
 
         tokens.forEachIndexed { index, token ->
             val latch = CountDownLatch(tokens.size)
-            JWTTools().verify(token) { err, payload ->
-                assertNull("token $index failed verification", err)
-                println(payload)
-                latch.countDown()
+            GlobalScope.launch {
```

Comment:
after the callback is replaced by return in the verify method this test code can be simplified a lot.

```
runBlocking {
  val payload = JWTTools().verify(token)
  assertNotNull(payload)
}
```

No more countdown latches